language/transaction-builder/generator/examples/golang/stdlib_demo.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 func main() {
-	token := &libra.TypeTag__Struct{
+	token := libra.TypeTag__Struct{
```

Comment:
as libra.TypeTag is an interface, we should pass in as ptr instead of value, so I suggest keep original &xxx version